From 19b683f075d11b920552990d16b9a7a82eed12e3 Mon Sep 17 00:00:00 2001
From: Andrea Bolognani <abologna@redhat.com>
Date: Mon, 12 Dec 2022 19:32:32 +0100
Subject: [PATCH] virt-install: Recommend '--boot uefi'

Firmware autoselection is the way to go in most cases, so
recommend that instead of telling users that they should provide
all information manually.

Signed-off-by: Andrea Bolognani <abologna@redhat.com>
(cherry picked from commit f2b5aaf458764ec7ecf105038e5f2f7cc26b6c17)

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2112154

Signed-off-by: Jonathon Jongsma <jjongsma@redhat.com>
---
 man/virt-install.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/man/virt-install.rst b/man/virt-install.rst
index 3a6e8dcd2..684f22655 100644
--- a/man/virt-install.rst
+++ b/man/virt-install.rst
@@ -955,13 +955,13 @@ Some examples:
     Configure the VM to boot from UEFI. In order for virt-install to know the
     correct UEFI parameters, libvirt needs to be advertising known UEFI binaries
     via domcapabilities XML, so this will likely only work if using properly
-    configured distro packages.
+    configured distro packages. This is the recommended UEFI setup.
 
 ``--boot loader=/.../OVMF_CODE.fd,loader.readonly=yes,loader.type=pflash,nvram.template=/.../OVMF_VARS.fd,loader_secure=no``
     Specify that the virtual machine use the custom OVMF binary as boot firmware,
     mapped as a virtual flash chip. In addition, request that libvirt instantiate
     the VM-specific UEFI varstore from the custom "/.../OVMF_VARS.fd" varstore
-    template. This is the recommended UEFI setup, and should be used if
+    template. This setup is not recommended, and should only be used if
     --boot uefi doesn't know about your UEFI binaries. If your UEFI firmware
     supports Secure boot feature you can enable it via loader_secure.
 
-- 
2.39.1

